src/Api/WebClient.php

```diff
@@ -146,14 +148,14 @@ public function createShipment(ShipmentInterface $shipment): array
             'parcels' => [$this->createParcel($shipment)],
             'service' => $this->getShippingGatewayConfig('service'),
             'additional_services' => $this->getAdditionalServices(),
-            'reference' => "Order: ".$order->getNumber(),
+            'reference' => 'Order: ' . $order->getNumber(),
             'comments' => $this->resolveComment($order),
             'is_return' => $this->shippingGateway->getConfigValue('is_return'),
         ];
 
-        if (!empty($this->getShippingGatewayConfig('insurance_amount'))) {
+        if (null !== $this->getShippingGatewayConfig('insurance_amount')) {
```

Comment:
Nice. We could put it into separate variable and reuse it in the line 158, but it's not a must